op-guide/configuration.md

```diff
@@ -94,6 +94,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: true
 + The value can be (true) or (false). (true) is to enable join without any equal conditions. (false) is to disable it.
 
+### --proxy-protocol-networks
++ PROXY Protocol allowed proxy server IP addresses or CIDRs (split by `,`).
++ default: "" (empty string)
++ The value can be IP address (192.168.1.50) or CIDR (192.168.1.0/24), if require more than one address use `,` to split. `*` means any IP addresses. Leaving it empty disable PROXY Protocol.
+
+### --proxy-protocol-header-timeout
++ PROXY Protocol header read timeout.
++ default: 5 (seconds)
++ The value set timeout for PROXY protocol header read. Unit is second. You should not set this value to 0.
```

Comment:
Can we change it to: The value sets the timeout for the PROXY protocol header read. The unit is second. You cannot set this value to 0.

Reply:
Actuarially the timeout can be set to 0, and this sometimes may cause strange issue, and most time it can be work correctly. So `should not` is more fit here.